src/Weather/Skysight/APIQueue.hpp

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright_License {
+
+  XCSoar Glide Computer - http://www.xcsoar.org/
+  Copyright (C) 2000-2016 The XCSoar Project
+  A detailed list of copyright holders can be found in the file "AUTHORS".
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+}
+*/
+#ifndef WEATHER_SKYSIGHTAPI_QUEUE_HPP
+#define WEATHER_SKYSIGHTAPI_QUEUE_HPP
+
+#include "Request.hpp"
+#include "Metrics.hpp"
+#include "Event/Timer.hpp"
+#include <vector>
+
+class SkysightAPIQueue final: public Timer {
+  std::vector<std::unique_ptr<SkysightAsyncRequest>> request_queue;
+  bool is_busy = false;
+
+  void Process();
```

Comment:
Bad coding style

Reply:
Which of is_busy or Process(0 are youi

Reply:
Which of is_busy or Process() do you mention?

Reply:
Mixing fields and methods.

Reply:
True. Move Process() and OnTimer() below tstring password.